index.html

```diff
@@ -3,45 +3,65 @@
 <html lang="en">
 
 <head>
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=GA_MEASUREMENT_ID"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag() { window.dataLayer.push(arguments); }
+    gtag('js', new Date());
+
+    gtag('config', 'G-G1FFR90YL2');
+  </script>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <link rel="stylesheet" href="style.css">
   <title>Amel Portfolio</title>
 </head>
 
-<body>
+<body tabindex="0">
   <main>
-  
+
     <header class="header-class">
       <nav class="nav-container">
-        <a href="#" class="flex-logo"><img src="./img/My-Logo.png" alt="logo"></a>
-        <div class="flex-menu"><img src="./img/Menu-button.png" alt="menu"></div>
+        <a href="#" class="flex-logo">AMEL</a>
+        <div id="menu-toggle" class="flex-menu"><img src="./img/Menu-button.png" alt="menu"></div>
         <ul class="desktop-toolbar">
-          <li><a href="#">Portfolio</a></li>
-          <li><a href="#">About</a></li>
-          <li><a href="#">Contact</a></li>
+          <li><a href="#portfolio">Portfolio</a></li>
+          <li><a href="#about">About</a></li>
+          <li><a href="#contact">Contact</a></li>
         </ul>
       </nav>
```

Comment:
[ OPTIONAL ]The hamburger menu is aligned left on the tablet viewport. It should cover the full-screen width and follow the design. Kindly see the screenshot below and fix the width issue.
![tablet](https://user-images.githubusercontent.com/35504149/132393364-e21b7023-9b29-485d-ae6d-b876d9322749.png)